src/main/java/graphql/util/querygenerator/QueryGeneratorOptions.java

```diff
@@ -0,0 +1,82 @@
+package graphql.util.querygenerator;
+
+import com.google.common.base.Predicates;
```

Comment:
We have a checker in the build to look for new Guava imports, and we prefer to not add new imports.

Is it possible to use the plain Java version of this instead?